### DISCUSSION TITLE
About Unknown MS Compiler version 19xx

### DISCUSSION THREAD

**Discussion by imba-tjd** [pypa/setuptools#2675] [General]
Some developers try to use MinGW/MinGW-w64, but they will get an error like `ValueError: Unknown MS Compiler version 19xx`.

There is an issue and a patch in https://bugs.python.org/issue25251

Cython says don't use MinGW-w64: https://github.com/cython/cython/wiki/CythonExtensionsOnWindows#less-useful-information

https://github.com/numpy/numpy/pull/6875 says `vcruntime140.dll does not contain the C runtime`

Any thoughts?

Edit: I created https://github.com/pypa/distutils/pull/41

**Comment by webknjaz** [comment]
I think a pull request implementing this would be welcome.
